### PR TITLE
Add multinode-tls docker environment

### PR DESCRIPTION
**Pull request by arhimondr**

### CODE REVIEW

File: presto-product-tests/conf/docker/multinode-tls/compose.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+source ${BASH_SOURCE%/*}/../common/compose-commons.sh
```
Comment (ebd2) [inline presto-product-tests/conf/docker/multinode-tls/compose.sh]: Refactor the ${BASH_SOURCE%/*} into a common variable

**Comment by ebd2** [comment]
LGTM % nit.

**Comment by maciejgrzybek** [comment]
Looks good. I'm not the best person to review docker infra though.

**Comment by ArturGajowy** [comment]
Why do we need this?

**Comment by ArturGajowy** [comment]
Why two presto worker nodes, instead of 1 (as in multinode)?

**Comment by ArturGajowy** [comment]
Did you manage to run product tests with that @ Travis?

**Comment by arhimondr** [comment]
> Why do we need this?

This environment is needed to test the secured (by TLS+LDAP/KERBEROS) internal communication between the Presto nodes.

> Why two presto worker nodes, instead of 1 (as in multinode)?

To make sure that `WORKER -> WORKER` communication is possible. With only 2 nodes (`COORDINATOR`, `WORKER`) the only we can check it is the `COORDINATOR -> WORKER` and `WORKER -> COORDINATOR`  communication.

> Did you manage to run product tests with that @ Travis?

No, we didn't. We don't have the actual test suite for this environment yet. We are going to work on this on a very next sprint.

**Pull request merged**